ludwig/utils/visualization_utils.py

```diff
@@ -156,7 +156,10 @@ def compare_classifiers_plot(
     width = 0.8 / num_metrics if num_metrics > 1 else 0.4
     ticks = np.arange(len(scores[0]))
 
-    colors = plt.get_cmap("tab10").colors
+    if num_metrics <= 10:
```

Comment:
Is there any reason we couldn't default to `"tab20"` in all three locations?

Reply:
That makes sense given the way the two palettes are organized. Thanks for the clarification!